src/models/childModel.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const childSchema = new mongoose.Schema({
+    name: {
```

Comment:
don't we also require username here? and maybe password too
so that children can log in

Reply:
So we will have front end change this for child password.